pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -48,8 +48,8 @@ import (
 )
 
 const (
-	// If the pod stays in unschedulableQ longer than the unschedulableQTimeInterval,
-	// the pod will be moved from unschedulableQ to activeQ.
+	// If a pod stays in unschedulableQ longer than unschedulableQTimeInterval,
+	// the pod will be moved from unschedulableQ to backoffQ or activeQ.
```

Comment:
Nit: the variable name is `podBackoffQ` (just so it's consistent with other queue names used here)?

Reply:
The term `backoffQ` is used a lot in the codebase and IMO it's not vague.